Docs/Reference/Functions/draw_set_align.md

```diff
@@ -0,0 +1,33 @@
+[Home](/README.md) > [Community Toolbox Reference](/Docs/Reference/Reference.md) > [Drawing utilities](/Docs/Reference/Groups/DrawingUtils.md) > [Settings](/Docs/Reference/Groups/Drawing:Settings.md)
+
+# draw_set_align
+
+[Go to source](/Community%20Toolbox/scripts/utils_CommunityToolboxDrawing/utils_CommunityToolboxDrawing.gml#L27)
+
+Sets the horizontal and vertical alignment of the drawn text.
+
+###### Arguments
+
+| Argument | Type | Usage | Description |
+|:---:|:---:|:---:|:---|
+| **halign** | **`Constant.HAlign`** | **Required** | The horizontal alignment to set. |
+| **valign** | **`Constant.VAlign`** | **Required** | The vertical alignment to set. |
+
+###### Returns
+N/A
+
+### Example
+
+The following code draws text in the middle of a button.
+
+```gml
+draw_self();
+draw_set_color_alpha(c_yellow, 1);
+draw_set_align(fa_center, fa_middle);
+draw_set_font(fnt_Button);
+draw_text(x + sprite_width div 2, y + sprite_height div 2, "Click me!");
+```
```

Comment:
Minor nit: In the `draw_set_color_alpha` example on the docs page, you set color and alpha back to the "default" of `c_white` and `1.0`. Here, you don't reset either the color/alpha or the text alignment. Might be good for the documentation to encourage a consistent style on that.

Reply:
Added resetting the colour to the other example, but not resetting align, because I tend to set the align every time I draw some text anyway. In contrast, when I didn't reset the colour or alpha, weird things happened with drawn sprites and stuff, so now I make sure to reset properly.